### PR TITLE
hal: cli script for bulk push

### PR DESCRIPTION
**Pull request by puntonim**

### CODE REVIEW

File: inspirehep/modules/hal/cli.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+#
+# In applying this license, CERN does not waive the privileges and immunities
+# granted to it by virtue of its status as an Intergovernmental Organization
+# or submit itself to any jurisdiction.
+
+from __future__ import absolute_import, division, print_function
+
+import click
+
+from flask_cli import with_appcontext
```
Comment (drjova) [inline inspirehep/modules/hal/cli.py]: ```flask.cli  import with_appcontext```, ```flask_cli``` has been deprecated :)

File: inspirehep/modules/hal/bulk_push.py [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+from flask import current_app
+
+from invenio_records.models import RecordMetadata
+from inspirehep.modules.hal.core.tei import convert_to_tei
+from inspirehep.modules.hal.core.sword import create, update
+
+
+def _set_config():
+    # Set the proper configuration.
+    current_app.config['RECORDS_SKIP_FILES'] = True
```
Comment (jacquerie) [inline inspirehep/modules/hal/bulk_push.py]: This is unneeded.

File: inspirehep/modules/hal/cli.py (hunk 1 above)
Comment (puntonim) [reply inspirehep/modules/hal/cli.py]: Done

File: inspirehep/modules/hal/bulk_push.py (hunk 2 above)
Comment (puntonim) [reply inspirehep/modules/hal/bulk_push.py]: Done

**Comment by drjova** [comment]
If we'll keep the script for a while, IMO would be safer to use `` click.echo`` instead of ``print``. WDYT?

**Comment by puntonim** [comment]
done

**Pull request merged**